notifyall/notifyall.php

```diff
@@ -70,23 +66,20 @@ function notifyall_post(App $a)
 
 	// if this is a test, send it only to the admin(s)
 	// admin_email might be a comma separated list, but we need "a@b','c@d','e@f
-	if (intval($_REQUEST['test'])) 
-	{
+	if (intval($_REQUEST['test'])) {
 		$email = $a->config['admin_email'];
 		$email = "'" . str_replace([" ",","], ["","','"], $email) . "'";
 	}
 	$sql_extra = ((intval($_REQUEST['test'])) ? sprintf(" AND `email` in ( %s )", $email) : '');
 
 	$recips = q("SELECT DISTINCT `email` FROM `user` WHERE `verified` AND NOT `account_removed` AND NOT `account_expired` $sql_extra");
 
-	if (! $recips) 
-	{
+	if (! $recips)  {
```

Comment:
You're going to laugh, but there is one space too many before the opening brace.